ev3dev2/motor.py

```diff
@@ -2182,6 +2189,138 @@ def turn_left(self, speed, degrees, brake=True, block=True):
         """
         self._turn(speed, abs(degrees) * -1, brake, block)
 
+    def odometry_coordinates_log(self):
+        log.info("%s: odometry angle %s at (%d, %d)" %
```

Comment:
Do you know what loop times we get on the EV3?

Reply:
Testing now on a pi3 and with no sleep at all there are normally 2ms to 5ms between lines of log output.  Will test later on EV3.

Reply:
On an EV3 it is hard to tell...just the log.info() call adds 30ms :(

Reply:
2-5ms seems reasonable given the constraints... RE: the thread about sleep time, I do feel like the time should be shorter, but if it works as-is that isn't really an issue.